.docker/Dockerfile

```diff
@@ -0,0 +1,115 @@
+# syntax = docker/dockerfile:1.3
+
+# ghcr.io/ros-planning/moveit2:${ROS_DISTRO}-source
+# Downloads the moveit source code and install remaining debian dependencies
+
+ARG ROS_DISTRO=rolling
+
+######################### Base Tutorial Image  #########################################
+
+FROM moveit/moveit2:${ROS_DISTRO}-source as base_image
+
+# Export ROS_UNDERLAY for downstream docker containers
+ENV ROS_UNDERLAY /root/ws_moveit/install
+WORKDIR $ROS_UNDERLAY/..
```

Comment:
This is part of the `source` image:
https://github.com/ros-planning/moveit2/blob/2de48a7aafacd45d4f9d4c667ff7337b1da31ddd/.docker/source/Dockerfile#L10-L13
```suggestion
```